src/ivoc/scene.cpp

```diff
@@ -536,7 +532,7 @@ void Scene::request(Requisition& req) const {
 
 void Scene::allocate(Canvas* c, const Allocation& a, Extension& ext) {
     // printf("Scene::allocate\n");
-    for (GlyphIndex index = 0; index < info_->size(); ++index) {
+    for (const auto&& [index, _]: enumerate(*info_)) {
```

Comment:
This has me worrying that enumerate returns the item by value. For that reason I prefer the previous version.

Reply:
I think you have `range` for this?